lang/en/home.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Home Language Lines
+    |--------------------------------------------------------------------------
+    |
+    |
+    */
+
+    'our helping' => 'Our Helping',
```

Comment:
```suggestion
    'Our helping' => 'Our Helping',
```


Or is it required to be in lowercase ?